src/elements/content-sidebar/activity-feed/task-new/TaskCompletionRuleIcon.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { TASK_COMPLETION_RULE_ANY } from '../../../../constants';
+import messages from './messages';
+import Tooltip from '../../../../components/tooltip';
+import IconAnyTask from '../../../../icons/general/IconAnyTask';
+import type { TaskCompletionRule } from '../../../../common/types/tasks';
+
+import './TaskCompletionRuleIcon.scss';
+
+type Props = {|
+    completionRule: ?TaskCompletionRule,
+|};
+
+const TaskCompletionRuleIcon = ({ completionRule }: Props): React.Node => (
+    <React.Fragment>
```

Comment:
It seems like the condition should live in the parent, but if not, this can still be simplified to something like the following:

```
const TaskCompletionRuleIcon = ({ completionRule }: Props): React.Node => ( 
    completionRule === TASK_COMPLETION_RULE_ANY &&  (
        <Tooltip position="top-center" text={<FormattedMessage {...messages.taskAnyAffordanceTooltip} />}>
            <IconAnyTask height={11} width={11} className="bcs-TaskCompletionRuleIcon" />
        </Tooltip>
    )
);
```

Reply:
When we add new completion rules in the future, then all the logic to decide what icon to render can live in this tooltip